jpa/subsystem/src/main/java/org/jboss/as/jpa/interceptor/WebNonTxEmCloserAction.java

```diff
@@ -46,7 +46,7 @@ public void setup(final Map<String, Object> properties) {
 
     @Override
     public void teardown(final Map<String, Object> properties) {
-        NonTxEmCloser.popCall();    // close any transactional entity managers that were created without a jta transaction.
+        NonTxEmCloser.popCall();    // close any transactional entity managers that were created without Jakarta Transactions.
```

Comment:
Missing indefinite article (an `a`).

Reply:
I agree with @mmusgrov, how about one of:
1. `close any transactional entity managers that were created without a Jakarta Transactions transaction.`
2. `close any transactional entity managers that were created without an active Jakarta Transactions transaction.`
3. `close any transactional entity managers that were created without a (Jakarta Transactions) transaction.`
4. `close any transactional entity managers that were created without an active (Jakarta Transactions) transaction.`

Reply:
went with 1 as going forward with `jta transaction` to `Jakarta Transactions transaction`.